metadata-ingestion/tests/integration/lookml/test_lookml.py

```diff
@@ -1265,8 +1266,20 @@ def test_unreachable_views(pytestconfig):
         LookMLSourceConfig.parse_obj(config),
         ctx=PipelineContext(run_id="lookml-source-test"),
     )
-    wu: List[MetadataWorkUnit] = [*source.get_workunits_internal()]
-    assert len(wu) == 15
+    workunits: List[Union[MetadataWorkUnit, Entity]] = [
+        *source.get_workunits_internal()
+    ]
+    converted_workunits: List[MetadataWorkUnit] = []
+    # Convert entities to metadata work units,
+    for workunit in workunits:
+        if isinstance(workunit, Entity):
+            converted_workunits.extend(workunit.as_workunits())
+        else:
+            converted_workunits.append(workunit)
```

Comment:
This may be a matter of taste...

```
    converted_workunits: List[MetadataWorkUnit] = [
        wu.as_workunits() if isinstance(wu, Entity) else wu
        for wu in workunits
    ]
```